sql/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/ArtifactSuite.scala

```diff
@@ -199,10 +199,12 @@ class ArtifactSuite extends ConnectFunSuite with BeforeAndAfterEach {
   }
 
   test("Batched SingleChunkArtifacts") {
-    val file1 = artifactFilePath.resolve("smallClassFile.class").toUri
-    val path = artifactFilePath.resolve("smallJar.jar")
-    val file2 = path.toUri
-    assume(path.toFile.exists)
+    val path1 = artifactFilePath.resolve("smallClassFile.class")
+    val file1 = path1.toUri
+    val path2 = artifactFilePath.resolve("smallJar.jar")
+    val file2 = path2.toUri
+    assume(path1.toFile.exists)
```

Comment:
Why not put `assume(path1.toFile.exists)` between `path1` and `file1` ?

Reply:
yeah let's do to be consistent